pkgs/by-name/ne/nexusmods-app/package.nix

```diff
@@ -140,15 +141,23 @@ buildDotnetModule (finalAttrs: {
       let
         runTest =
           name: script:
-          runCommand "${pname}-test-${name}" { nativeBuildInputs = [ finalAttrs.finalPackage ]; } ''
-            ${script}
-            touch $out
-          '';
+          runCommand "${pname}-test-${name}"
+            {
+              nativeBuildInputs = [ finalAttrs.finalPackage ];
+              FONTCONFIG_FILE = makeFontsConf {
+                fontDirectories = [ ];
+              };
```

Comment:
If we just want an empty and/or default fontconfig file, is there a better alternative to using `makeFontsConfig`?